From fdf2767e8dc54727c9536a4d39d230a959e3698c Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 2 Apr 2020 16:16:14 +0000
Subject: [PATCH] GCC: add missing apps namespace to BrowserAppLauncher

GCC does not get namespace of BrowserAppLauncher right and
fails like this:

chrome/browser/apps/app_service/app_service_proxy.h:82:23: error:
declaration of 'apps::BrowserAppLauncher&
apps::AppServiceProxy::BrowserAppLauncher()' changes meaning of
'BrowserAppLauncher' [-fpermissive]
---
 chrome/browser/apps/app_service/app_service_proxy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/apps/app_service/app_service_proxy.h b/chrome/browser/apps/app_service/app_service_proxy.h
index 1ebadfd..83f9c71 100644
--- a/chrome/browser/apps/app_service/app_service_proxy.h
+++ b/chrome/browser/apps/app_service/app_service_proxy.h
@@ -79,7 +79,7 @@ class AppServiceProxy : public KeyedService,
   apps::InstanceRegistry& InstanceRegistry();
 #endif
 
-  BrowserAppLauncher& BrowserAppLauncher();
+  apps::BrowserAppLauncher& BrowserAppLauncher();
 
   apps::PreferredApps& PreferredApps();
 
-- 
2.24.1

